Main1.cpp

```diff
@@ -6,6 +6,7 @@
 #include "Square.cpp"
 #include "Rectangle.cpp"
 #include "Circle.cpp"
+#include "Triangle.cpp"
```

Comment:
Why are we including `Triangle.cpp` in our main file/function? What purpose does it serve?

Reply:
removed all the unnecessary includes in main 1

Reply:
https://github.com/Muhammadali2025222/C-Practice/pull/11/files check this to confirm changes